assets/l10n/app_en.arb

```diff
@@ -552,6 +552,24 @@
       "details": {"type": "String", "example": "type 'Null' is not a subtype of type 'String' in type cast"}
     }
   },
+  "errorMalformedResponseFormat": "{className}: {httpStatus} {routeName}: {message}",
+  "@errorMalformedResponseFormat": {
+    "description": "Format for error messages from malformed server responses. The message parameter is already localized.",
```

Comment:
nit: the audience of the string description is mainly the translators; so there is nothing actionable for them knowing that `message` is localized. Similarly, we can leave out "format" in both the name of the string and its description.

How about naming it "errorMalformedServerResponse"? For rewriting the description, you can find soem examples with:

`git grep -hEA2 'error.+\",' assets/l10n/app_en.arb`

You can check out `git grep --help` for what each of these options mean.